flex-template-future-cc-demo/public/appConfig.example.js

```diff
@@ -8,8 +8,9 @@ var appConfig = {
   showSupervisorDesktopView: true,
   custom_data: {
     serverless_functions_protocol: 'http',
```

Comment:
What's the reason that this property didn't get renamed? It is also overriding the PS Template features from --included-remote.